dll/win32/kernel32/include/kernel32.h

```diff
@@ -50,6 +50,8 @@
 #define ROUND_UP(n, align) \
     ROUND_DOWN(((ULONG)n) + (align) - 1, (align))
 
+#define ARRAY_SIZE(a) (sizeof(a)/sizeof((a)[0]))
```

Comment:
Can we move that into the concerned `/wine/` .c source files?

Reply:
But lang.c is laid in /winnls/ subfolder, not in /wine/